geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml

```diff
@@ -22,12 +22,13 @@
         <dependencySet>
             <outputDirectory>dist/gs-plugins</outputDirectory>
             <directoryMode>0755</directoryMode>
-            <fileMode>0755</fileMode>
+            <fileMode>0644</fileMode>
             <useProjectArtifact>false</useProjectArtifact>
             <useTransitiveDependencies>false</useTransitiveDependencies>
             <useStrictFiltering>true</useStrictFiltering>
             <includes>
                 <include>org.locationtech.geomesa:geomesa-hbase-gs-plugin*</include>
+                <include>org.locationtech.geomesa:geomesa-hbase-process*</include>
```

Comment:
Hmm, I would have to dial in my maven foo. I'm using the hbase project to create an install.tar.gz so it matches Accumulo and so users will already be used to doing it. I guess we could have geomesa-process create the jar and a tar and import that here.